tutorials/math/random_number_generation.rst

```diff
@@ -166,6 +267,39 @@ more logic to this method::
         # The function will never return the same fruit more than once in a row.
         return random_fruit
 
+ .. code-tab:: csharp
+
+    string[] fruits = { "apple", "orange", "pear", "banana" };
+    string LastFruit = "";
+
+    public override void _Ready()
+    {
+        GD.Randomize();
+
+        foreach (int i in GD.Range(100))
+        {
+            // Pick 100 fruits randomly.
+            GD.Print(Get_Fruit()); 
+        }
+    }
+
+    public string Get_Fruit()
+    {
+        var RandomFruit = fruits[GD.Randi() % fruits.Length];
+        while (RandomFruit == LastFruit)
+        {
+            // The last fruit was picked, try again until we get a different fruit.
+            RandomFruit = fruits[GD.Randi() % fruits.Length];
```

Comment:
```suggestion
            randomFruit = _fruits[GD.Randi() % _fruits.Length];
```

[→ Naming conventions](https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/c_sharp_style_guide.html#naming-conventions)

I'd personally use a `do-while` here to avoid repeating code...